tests/expected_index_document.json

```diff
@@ -0,0 +1,166 @@
+                "state": "MA",
+                "street_address": "415 Main St."
+            },
+            "contributor": "Karthik,,Shekhar",
+            "geo_parent_series": "GSE81905",
+            "geo_series": "GSE81904",
+            "last_update_date": "2017-05-26",
+            "ncbi_bioproject": "PRJNA323370",
+            "overall_design": "Drop-Seq was performed on two different batches. Bipolar1-Bipolar4 are replicates from batch 1 and Bipolar5-6 are replicates from Batch 2",
+            "pmid": "27565351",
+            "release_status": "Public on Aug 25 2016",
+            "sra_project": "SRP075719",
+            "submission_date": "2016-05-25",
+            "summary": "Vsx2-GFP mouse retinas were dissected, FACS sorted for GFP+ cells and single-cell mRNAseq libraries generated with Drop-Seq",
+            "supplementary_files": [
+                "ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/series/GSE81904/GSE81904_BipolarUMICounts_Cell2016.txt.gz",
+                "ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/series/GSE81904/GSE81904_RAW.tar",
+                "ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/series/GSE81904/GSE81904_bipolar_data_Cell2016.Rdata.gz",
+                "ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/series/GSE81904/GSE81904_class.R.gz",
+                "ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/SRP075/SRP075719"
+            ],
+            "title": "Drop-Seq analysis of  P17 FACS sorted retinal cells from the Tg(Chx10-EGFP/cre,-ALPP)2Clc or Vsx2-GFP transgenic line",
+            "uuid": "f35c2bce-a4a7-4e1e-a8a6-8637e65d19a7"
+        },
+        "sample.json": {
+            "body_part": {
+                "name": "retina",
+                "ontology": "UBERON:0000966",
+                "organ": "eye"
+            },
+            "donor": {
+                "age": "17",
+                "age_unit": "day",
+                "genotype": "Vsx2-GFP",
+                "life_stage": "postpartum",
+                "ncbi_taxon": "10090",
+                "species": "Mus musculus"
+            },
+            "geo_sample": "GSM2177570",
+            "last_update_date": "2016-08-25",
+            "long_label": "P17 Vsx2-GFP sorted mouse bipolar retina cells 1 (Batch 1)",
+            "ncbi_biosample": "SAMN05177285",
+            "protocols": [
+                {
+                    "description": "suspensions were processed through Drop-Seq to generate single-cell cDNA libraries attached to microbeads. Microbeads were counted, and amplified by PCR, and the 3' end of the cDNA prepared for sequencing using a modified Nextera XT protocol.",
+                    "type": "nucleic acid library construction protocol"
+                }
+            ],
+            "short_label": "retina bipolar 1",
+            "submission_date": "2016-05-25",
+            "uuid": "5aca931f-9c88-4483-ba46-685b70c08a73"
+        }
+    }
+}
```

Comment:
For all of these data bundle files, I suggest that you pull them from the submodule instead of copying them here.